dbcon/joblist/rowestimator.cpp

```diff
@@ -238,7 +248,10 @@ float RowEstimator::estimateOpFactor(const T& min, const T& max, const T& value,
         case COMPARE_NLE:
             if (cpStatus == BRM::CP_VALID)
             {
-                factor = (1.0 * max - value) / (1.0 * max - min + 1);
+                if (!datatypes::Decimal::isWideDecimalType(ct))
+                    factor = (1.0 * max - value) / (1.0 * max - min + 1);
+                else
+                    factor = (1.0 * max - value) / (1.0 * max - min + 1);
```

Comment:
I presume here math works w/o an explicit cast to float128?

Reply:
Good catch. I missed the cast to __float128 here and in the next case, fixing this.